openprescribing/common/utils.py

```diff
@@ -63,6 +63,27 @@ def get_env_setting(setting, default=None):
             raise ImproperlyConfigured(error_msg)
 
 
+def get_env_setting_bool(setting, default=None):
```

Comment:
Does a default of `None` make sense for a boolean setting?

Reply:
In this context `None` means "no default supplied so blow up if the var isn't defined", which I just copied from the existing API

Reply:
Oh yeah, of course.